### DISCUSSION TITLE
zsh-autoswitch-virtualenv

### DISCUSSION THREAD

**Discussion by Pakrohk** [zimfw/zimfw#467] [Q&A] [answered]
I tried to install this module
`zmodule MichaelAquilina/zsh-autoswitch-virtualenv -n zsh_auto_env`
But after installation, the speed of running zsh has decreased greatly.
I tried to increase the speed of zsh with the zimfw compile command, but I got the following warning.

>! /home/`UserName`/.zimrc:58:zsh_auto_env: Nothing found to be initialized. Customize the module name or initialization with zmodule options.

Do you have a solution for this problem?

**Comment by bgrrtt** [comment, marked as answer]
Try `zmodule MichaelAquilina/zsh-autoswitch-virtualenv -n autoswitch_virtualenv`
